models/protocol_tests/generate.go

```diff
@@ -592,6 +592,8 @@ func getType(t string) uint {
 func main() {
 	fmt.Println("Generating test suite", os.Args[1:])
 	out := generateTestSuite(os.Args[1])
+
+	// Todo: fix test suites
 	if len(os.Args) == 3 {
```

Comment:
probably don't need this `todo` A tracking Github issue would be a better way to track the protocol tests no longer can be generated.